pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -102,14 +104,14 @@ public class SegmentColumnarIndexCreator implements SegmentCreator {
   private Schema _schema;
   private File _indexDir;
   private int _totalDocs;
-  private int _docIdCounter;
+  private int _docPosOnDisk;
```

Comment:
Don't change this. We should not update this when building the segment in column major fashion

Reply:
Done